### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by wheninseattle**
## What

Create home page layout including hero section, mission, and cards linking to partnership, volunteering and support.

## Why Do

Implementation of design for new home page.

## How Do

To-do...

## Show Me

Desktop:
image

Mobile:
image

### CODE REVIEW

File: pages/home.tsx [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+  const theme = useTheme()
+  const palette = theme.palette
+
+  const loader = ({ src, width, quality }) => {
+    return `${process.env.NEXT_PUBLIC_BASE_PATH}${src}\?w=${width}&q=${
+      quality || 75
+    }`
+  }
+
+  const isMediumScreen = useMediaQuery('(max-Width:900px)')
```
Comment (santina) [inline pages/home.tsx:29]: Hey I noticed a lot of css is made to make the website responsive. Could you look at [stack]( https://mui.com/material-ui/react-stack/) and [grid](https://mui.com/material-ui/react-grid2/)
could automatically adjust for screen size and solve the issue. 


Especially grid, here's a [video](https://youtu.be/jrgiTFwvglw?t=164) on it.

**Review by santina — COMMENTED** [review]
For some reason I don't see the image and it takes up the entire screen - people may not know to scroll down to see more content. Do you see similar issue on your end?  I totally get this is a draft so makes sense it's not ready just yet (I'm just excited to see it in action!) 

image

File: pages/home.tsx (hunk 1 above)
Comment (santina) [reply pages/home.tsx:29]: oh like what @mistyb01  did [here](https://github.com/openseattle/open-seattle-website/pull/18/files#diff-7321353cedfba627ec35e731b0b944c9cdca5110bd271896aab3ba468a51b1a5R75)

**Comment by santina** [comment]
image

**Comment by santina** [comment]
can't figure out the image. we can just use  for now. you can add `/* eslint-disable @next/next/no-img-element */` at the beginning of the file

File: pages/home.tsx (hunk 1 above)
Comment (wheninseattle) [reply pages/home.tsx:29]: Thanks Santina - yeah, it does feel like a lot of CSS. I was struggling to get all the graphics and the shadows to work with the stack component. Is there a specific place you would recommend implementing it?

Comment (santina) [reply pages/home.tsx:29]: mm stack should only be a wrapper around the components and not impacting the components themselves. Could you share what you had and we can debug together?

**Comment by santina** [comment]
I copied and pasted the hero section to homepage_new and the formatting is not fitting together well. It seems like a lot of css is to make the gradient responsive to window size. 

Maybe try something like with the [CommonHeader](https://github.com/openseattle/open-seattle-website/blob/main/components/CommonHeader.tsx#L49) component? You can define two sets of UI, one appear when it's small window size, and disappear when it's bigger `display: { xs: 'flex', md: 'none' } }`, and taken over by another one that is `display: { xs: 'none', md: 'flex' }`. That way you can define them independently without having to worried about specifying window size and rotating. Might make it easier to understand and be maintained.

**Pull request merged**